mne_bids/tests/test_write.py

```diff
@@ -130,8 +132,13 @@ def _validate(bids_root):
 def _test_anonymize(raw, bids_path, events_fname=None, event_id=None):
     bids_root = _TempDir()
     bids_path = _bids_path.copy().update(root=bids_root)
+    if raw.info['meas_date'] is not None:
+        daysback, _ = get_anonymization_daysback(raw)
+    else:
+        # just pass back any arbitrary number if no measurement date
+        daysback = 3300
```

Comment:
Why is `daysback` now an order of magnitude smaller than before?

Reply:
That was added because if the measurement date is set, then use the function `get_anonymization_daysback`, but if it is not set, then all that is needed is some arbitrary number (e.g. it could be `1`).

This issue comes up in `test_write.py/test_set`, where the `.set` dataset has no measurement date, but you want to test anonymization.